build-tools/packages/build-cli/src/codeCoverage/README.md

```diff
@@ -0,0 +1,62 @@
+# Code-Coverage-Analysis-Tool
+
+## Overview
+
+This module contains all the utilities required to run code coverage analysis on PRs. The tool uses the coverage reports generated in the PR build to run comparison against a baseline CI build for packages that have been updated on the PR. If the tool finds that the line or branch coverage for a package has been impacted in the PR, it posts a comment on the PR showing the diff of the line coverage between baseline and PR.
+
+## Generating coverage reports
+
+The code coverage plugin only generates coverage reports when tests run. You can generate these reports for your package locally by running `npm run test:coverage` for the individual package or by running `npm run ci:test:mocha:coverage` from the root.
```

Comment:
Is there a `ci:test:coverage` script we should use instead? Maybe we should add it?

Reply:
Sorry didn't get the suggestion here. What is the need of a script here?